docs/contributing.rst

```diff
@@ -1 +1 @@
-.. include:: ../CONTRIBUTING.md
+.. mdinclude:: ../CONTRIBUTING.md
```

Comment:
This should fix the formatting.  went ahead and pushed since I couldn't comment

Reply:
verified that formatting is working now: https://pysat.readthedocs.io/en/v3-0-2rc/contributing.html